src/software/jetson_nano/thunderloop.h

```diff
@@ -134,5 +136,17 @@ class Thunderloop
     const double PACKET_TIMEOUT_NS = 500.0 * NANOSECONDS_PER_MILLISECOND;
 
     // Path to the CPU thermal zone temperature file
-    const std::string CPU_TEMP_FILE_PATH = "/sys/class/thermal/thermal_zone1/temp";
+    const std::string CPU_TEMP_FILE_PATH     = "/sys/class/thermal/thermal_zone1/temp";
+    const std::string PATH_TO_RINGBUFFER_LOG = "/var/log/dmesg";
 };
+
+/*
+ * reads from the kernel ring buffer, likely, /var/log/dmesg file to see if the power
+ * is stable
+ *
+ * This is not defined in Thunderscope because I want to test this function without
+ * injecting gtest into the private class in Thunderloop
```

Comment:
nit:
```suggestion
 * This is not defined in Thunderloop to allow it to be unit tested easily
```
Also, its probably fine if you had this function as a static public method as well.

Reply:
done.